src/js/blue-button/components/GlossaryModal.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Modal from '../../common/components/Modal';
+
+class GlossaryModal extends React.Component {
```

Comment:
Is this component necessary? The isVisible check seems redundant with the Modal component's internal check and use of the visible property, and the close handler could be moved up into the parent component.